src/components/Task/CreateNew/index.jsx

```diff
@@ -9,7 +9,7 @@ import { ATTRIBUTE_TITLE, BUTTON_TYPE_BUTTON } from "utils/const/form-elements";
 import { ALT_TAG_ICON_DELETE, ICON_DELETE } from "utils/const/images";
 import styles from "components/Task/CreateNew/index.module.scss";
 
-function CreateTask({ onAddTask }) {
+function CreateTask({ onAddTask, onDeleteClick }) {
```

Comment:
prefer consistent naming for callback props.

```suggestion
function CreateTask({ onAddTask, onDeleteTask }) {
```